notebooks/en/trl_grpo_reasoning_advanced_reward.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<p>you can remove torch and accelerate, these are dependencies of trl</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>